spec/models/document_type_selection_spec.rb

```diff
@@ -36,6 +36,13 @@
       ids = document_type_selections.pluck("id")
       expect(ids.size).to eq(ids.uniq.size)
     end
+
+    it "only allows unique options in each document_type_selection" do
+      document_type_selections.each do |document_type_selection|
+        ids = document_type_selection["options"].pluck("id")
+        expect(ids.size).to eq(ids.uniq.size)
```

Comment:
Minor: could we get a better error message if we lost the `size`?

Reply:
Have added an error message to the test.